sormas-api/src/main/java/de/symeda/sormas/api/labmessage/TestReportDto.java

```diff
@@ -44,6 +44,8 @@
 	private String testResultText;
 	@Size(max = COLUMN_LENGTH_TEXT, message = Validations.textTooLong)
 	private String typingId;
+	@Size(max = COLUMN_LENGTH_SMALL, message = Validations.textTooLong)
+	private String externalId;
```

Comment:
The externalId in other entities has max size COLUMN_LENGTH_DEFAULT, I think this one should also be the same

Reply:
In the databse, it is introduced as `varchar(255)`, so I think it should point to something of size 255. 
It seems that this was changed recently, I suggest to use [FieldContraints.CHARACTER_LIMIT_SMALL](https://github.com/hzi-braunschweig/SORMAS-Project/blob/0ae2c81f7e75bf4d69a261e3de3ed8ad59051420/sormas-api/src/main/java/de/symeda/sormas/api/utils/FieldConstraints.java#L7). Okay?

Reply:
It is introduced by the PR as `varchar(255)` so it can be changed and I think it's better to keep similar to other externalId fields that are `varchar(512)`